Jenkinsfile

```diff
@@ -11,9 +11,9 @@ pipeline {
         DOCKER_PASSWORD = "${env.DOCKER_CREDENTIALS_PSW}"
         KONG_PACKAGE_NAME = "kong"
         DOCKER_CLI_EXPERIMENTAL = "enabled"
-        PULP_HOST_PROD = "https://api.pulp.konnect-prod.konghq.com"
+        PULP_HOST_PROD = "https://api.download.konghq.com"
         PULP_PROD = credentials('PULP')
-        PULP_HOST_STAGE = "https://api.pulp.konnect-stage.konghq.com"
+        PULP_HOST_STAGE = "https://api.download-dev.konghq.com"
```

Comment:
Did we bring back the `dev` environment - I thought it no longer existed?

Reply:
We do have a dev environment in cloud02